gordo/machine/dataset/data_provider/ncs_reader.py

```diff
@@ -346,13 +346,16 @@ def read_tag_files(
                 if dry_run:
                     logger.info("Dry run only, returning empty frame early")
                     return pd.Series()
-
+                before_downloading = timeit.default_timer()
                 with adls_file_system_client.open(file_path, "rb") as f:
                     df = file_type.read_df(f)
                     df = df.rename(columns={"Value": tag.name})
+                    df.index = pd.to_datetime(df.index, utc=True)
                     df = df[~df["Status"].isin(remove_status_codes)]
                     all_years.append(df)
-                    logger.info(f"Done parsing file {file_path}")
+                    logger.info(
+                        f"Done parsing in {(timeit.default_timer()-before_downloading):.2f} seconds for file {file_path}"
```

Comment:
I suggest rewording this logging statement to clarify that it is both downloading and parsing that is being timed.